cast/src/starknet_commands/helpers.rs

```diff
@@ -0,0 +1,10 @@
+use starknet::core::types::FieldElement;
```

Comment:
let's move this to lib.rs, as in this directory we only store starknet commands, as a convention

Reply:
[move parse number to lib](https://github.com/software-mansion/protostar/pull/2138/commits/cc68f5b024e3e6ae3311446387db32ef6431f36b)